ext/pdo_firebird/firebird_driver.c

```diff
@@ -32,6 +32,428 @@
+	/* 010     */ 16, /* CHR_WHITE */
+	/* 011     */ 0,
+	/* 012     */ 0,
+	/* 013     */ 16, /* CHR_WHITE */
+	/* 014     */ 0,
+	/* 015     */ 0,
+	/* 016     */ 0,
+	/* 017     */ 0,
+	/* 018     */ 0,
+	/* 019     */ 0,
+	/* 020     */ 0,
+	/* 021     */ 0,
+	/* 022     */ 0,
+	/* 023     */ 0,
+	/* 024     */ 0,
+	/* 025     */ 0,
+	/* 026     */ 0,
+	/* 027     */ 0,
+	/* 028     */ 0,
+	/* 029     */ 0,
+	/* 030     */ 0,
+	/* 031     */ 0,
+	/* 032     */ 16, /* CHR_WHITE */
+	/* 033  !  */ 0,
+	/* 034  "  */ 8, /* CHR_QUOTE */
+	/* 035  #  */ 0,
+	/* 036  $  */ 4, /* CHR_IDENT */
+	/* 037  %  */ 0,
+	/* 038  &  */ 0,
+	/* 039  '  */ 8, /* CHR_QUOTE */
+	/* 040  (  */ 0,
+	/* 041  )  */ 0,
+	/* 042  *  */ 0,
+	/* 043  +  */ 0,
+	/* 044  ,  */ 0,
+	/* 045  -  */ 0,
+	/* 046  .  */ 0,
+	/* 047  /  */ 0,
+	/* 048  0  */ 38, /* CHR_DIGIT | CHR_IDENT | CHR_HEX */
+	/* 049  1  */ 38, /* CHR_DIGIT | CHR_IDENT | CHR_HEX */
+	/* 050  2  */ 38, /* CHR_DIGIT | CHR_IDENT | CHR_HEX */
+	/* 051  3  */ 38, /* CHR_DIGIT | CHR_IDENT | CHR_HEX */
+	/* 052  4  */ 38, /* CHR_DIGIT | CHR_IDENT | CHR_HEX */
+	/* 053  5  */ 38, /* CHR_DIGIT | CHR_IDENT | CHR_HEX */
+	/* 054  6  */ 38, /* CHR_DIGIT | CHR_IDENT | CHR_HEX */
+	/* 055  7  */ 38, /* CHR_DIGIT | CHR_IDENT | CHR_HEX */
+	/* 056  8  */ 38, /* CHR_DIGIT | CHR_IDENT | CHR_HEX */
+	/* 057  9  */ 38, /* CHR_DIGIT | CHR_IDENT | CHR_HEX */
+	/* 058  :  */ 0,
+	/* 059  ;  */ 0,
+	/* 060  <  */ 0,
+	/* 061  =  */ 0,
+	/* 062  >  */ 0,
+	/* 063  ?  */ 0,
+	/* 064  @  */ 0,
+	/* 065  A  */ 37, /* CHR_LETTER | CHR_IDENT | CHR_HEX */
+	/* 066  B  */ 37, /* CHR_LETTER | CHR_IDENT | CHR_HEX */
+	/* 067  C  */ 37, /* CHR_LETTER | CHR_IDENT | CHR_HEX */
+	/* 068  D  */ 37, /* CHR_LETTER | CHR_IDENT | CHR_HEX */
+	/* 069  E  */ 37, /* CHR_LETTER | CHR_IDENT | CHR_HEX */
+	/* 070  F  */ 37, /* CHR_LETTER | CHR_IDENT | CHR_HEX */
+	/* 071  G  */ 5, /* CHR_LETTER | CHR_IDENT */
+	/* 072  H  */ 5, /* CHR_LETTER | CHR_IDENT */
+	/* 073  I  */ 5, /* CHR_LETTER | CHR_IDENT */
+	/* 074  J  */ 5, /* CHR_LETTER | CHR_IDENT */
+	/* 075  K  */ 5, /* CHR_LETTER | CHR_IDENT */
+	/* 076  L  */ 5, /* CHR_LETTER | CHR_IDENT */
+	/* 077  M  */ 5, /* CHR_LETTER | CHR_IDENT */
+	/* 078  N  */ 5, /* CHR_LETTER | CHR_IDENT */
+	/* 079  O  */ 5, /* CHR_LETTER | CHR_IDENT */
+	/* 080  P  */ 5, /* CHR_LETTER | CHR_IDENT */
+	/* 081  Q  */ 5, /* CHR_LETTER | CHR_IDENT */
+	/* 082  R  */ 5, /* CHR_LETTER | CHR_IDENT */
+	/* 083  S  */ 5, /* CHR_LETTER | CHR_IDENT */
+	/* 084  T  */ 5, /* CHR_LETTER | CHR_IDENT */
+	/* 085  U  */ 5, /* CHR_LETTER | CHR_IDENT */
+	/* 086  V  */ 5, /* CHR_LETTER | CHR_IDENT */
+	/* 087  W  */ 5, /* CHR_LETTER | CHR_IDENT */
+	/* 088  X  */ 5, /* CHR_LETTER | CHR_IDENT */
+	/* 089  Y  */ 5, /* CHR_LETTER | CHR_IDENT */
+	/* 090  Z  */ 5, /* CHR_LETTER | CHR_IDENT */
+	/* 091  [  */ 0,
+	/* 092  \  */ 0,
+	/* 093  ]  */ 0,
+	/* 094  ^  */ 0,
+	/* 095  _  */ 65, /* CHR_IDENT | CHR_INTRODUCER */
```

Comment:
@sim1984, `CHR_IDENT|CHR_INTRODUCER` would be `68`, wouldn't it? Not sure if it should be `69` (including `CHR_LETTER`).

Reply:
CHR_IDENT | CHR_INTRODUCER would be 68, wouldn't it?

Yes, of course, this is my fault.

Not sure if it should be 69 (including CHR_LETTER).

No, it shouldn't.

Reply:
Thanks for the swift reply! I'm going to fix this.